pkgs/top-level/all-packages.nix

```diff
@@ -9761,6 +9761,10 @@ in
 
   leaps = callPackage ../development/tools/leaps { };
 
+  ### DEVELOPMENT / JAVA MODULES
+
+  javaPackages = recurseIntoAttrs (callPackage ./java-packages.nix { });
```

Comment:
Hmm, I don't see the lists you're speaking of. Can you point them out? It looks like `java-pacakges.nix` results in a big attrset full of attributes it inherits from other .nix files?